api/src/main/java/org/apache/cloudstack/api/response/VirtualMachineResponse.java

```diff
@@ -0,0 +1,116 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why do we need this class? Is it possible to use UserVmResponse or do we also have affected VR and system VMs?

Reply:
if storage scope is changed from Zone to Cluster, the hosts in other cluster will disconnect from the storage.
the running VMs (user vms, vr, system vms) on the hosts will be affected.

Reply:
yes, the response can return system VMs as well.